src/Entity/Ern41/NewReleaseMessage.php

```diff
@@ -197,9 +197,9 @@ public function setMessageHeader(\DedexBundle\Entity\Ern41\MessageHeaderType $me
      * A Composite containing details of one or more Parties relating to the reported MusicalWorks.
      *
      * @return self
-     * @param \DedexBundle\Entity\Ern41\PartyType $party
+     * @param \DedexBundle\Entity\Ern41\PartyListType $party
      */
-    public function addToPartyList(\DedexBundle\Entity\Ern41\PartyType $party)
+    public function addToPartyList(\DedexBundle\Entity\Ern41\PartyListType $party)
```

Comment:
this should be changed to a setter (like setReleaseList ot setResourceList) (same for dealList)